### PR TITLE
Update import ordering to match Google Style Guide

### PR DESCRIPTION
**Pull request by skoushan**
Google Java Style Guide (recently updated): https://google.github.io/styleguide/javaguide.html#s3.3.3-import-ordering-and-spacing
ImportOrder documentation for reference: http://checkstyle.sourceforge.net/config_imports.html#ImportOrder

Alternate approach to #4

### CODE REVIEW

File: src/main/resources/spotify_checks.xml [hunk 1]
```diff
@@ -164,11 +164,13 @@
         </module>
         <module name="OverloadMethodsDeclarationOrder"/>
         <module name="VariableDeclarationUsageDistance"/>
-        <module name="CustomImportOrder">
-            <property name="specialImportsRegExp" value="com.spotify"/>
-            <property name="sortImportsInGroupAlphabetically" value="true"/>
-            <property name="customImportOrderRules" value="STATIC###SPECIAL_IMPORTS###THIRD_PARTY_PACKAGE###STANDARD_JAVA_PACKAGE"/>
-        </module>
+        <module name="ImportOrder">
+            <property name="option" value="above"/>
```
Comment (mattnworb) [inline src/main/resources/spotify_checks.xml]: I think this should be `top`: http://checkstyle.sourceforge.net/property_types.html#importOrder

> Imports are ordered as follows:
> 
> All static imports in a single block.
> All non-static imports in a single block.

https://google.github.io/styleguide/javaguide.html#s3.3.3-import-ordering-and-spacing

**Comment by udoprog** [comment]
dup of #4?

**Comment by dflemstr** [comment]
@udoprog its an alternate approach to my PR

File: src/main/resources/spotify_checks.xml (hunk 1 above)
Comment (skoushan) [inline src/main/resources/spotify_checks.xml]: Right, thanks!

**Comment by udoprog** [comment]
@dflemstr They are trying to accomplish the same thing. IMO only #4 or #5 should stay open to center discussion and approach.

**Comment by pettermahlen** [comment]
Can @dflemstr and @skoushan please elaborate on how these two PRs are different, and what the benefits are of the different approaches? It would be great to get some high-level descriptions rather than having to try to parse the checkstyle directives.

**Comment by dflemstr** [comment]
@udoprog closed my one in favor of this one.

@pettermahlen my config used regexes which is way overkill, this config gives better error messages because it uses semantics instead of regex matching.

**Comment by pettermahlen** [comment]
👍

**Pull request merged**